### ISSUE TITLE
Deprecation warning with some dependencies !!

### ISSUE THREAD

**Issue by vasudev-sharma** [Cloud-CV/Fabrik#526]
I tried following the instructions while installing **Fabri**k on my local machine as given in `Readme.md`.  Everything worked fine but I was getting deprecation warning while performing the step 13(`RUN npm install`). A screenshot of this is attached below. 
<img width="1438" alt="Screenshot 2019-11-24 at 1 12 42 AM" src="https://user-images.githubusercontent.com/31419210/69484292-87b5be80-0e57-11ea-9c10-8b219272df28.png">

Though, I know it's not a issue. However, I think we should try updating the dependencies such that the build doesn't fail. This would help us in maintaining this repo and help us in resolving  any such future dependency problems.

**Comment by salonitayal** [comment]
@vs74  While performing `Step 13/16: RUN npm install `  deprecation warning occurred on my system too. I tried `sudo docker-compose up --build` again.

**Comment by vasudev-sharma** [comment]
@Saloni-Tayal , yeah! I agree with you these warning occurs in` Step 13/16:` and also the installation works. What i am trying to say is, to reduce maintenance in future, I think it would be a good idea to deal with the whatever possible deprecated dependencies.
